appveyor.yml

```diff
@@ -10,15 +26,13 @@ test_script:
 nuget:
   disable_publish_on_pr: true
 
-skip_tags: true
-
 artifacts:
   - path: 'src\**\*.nupkg'
 
-deploy:
-- provider: NuGet
-  api_key:
-    secure: DtVDH1ZfIR8E2oDLeSPxfn0WFXw3MCv4WhLkkQQl9LfEHswb5sdxdCYodNKhrKik
-  artifact: /.*\.nupkg/
-  on:
-    branch: master
+# deploy:
```

Comment:
why is this not needed anymore?

Reply:
The deploy step? That's just temporary until we discuss how we want to handle package management (e.g. who's key do we use, is there a NLog nuget org, etc.).

Right now the reasons were:
- The nuget api key is current expired, this breaks the build for master.
- I wanted to test the tagging without accidentally releasing to nuget after the merge into master.

Reply:
OK thanks!



> * The nuget api key is current expired, this breaks the build for master.

Oops, we need a new issue for that.